src/databases/VTK/avtVTKWriter.C

```diff
@@ -96,13 +98,16 @@ double avtVTKWriter::INVALID_TIME = -DBL_MAX;
 //    Kathleen Biagas, Fri Feb 17 15:41:33 PST 2017
 //    Handle new Write options.
 //
+//    Mark C. Miller, Tue Apr  9 18:46:02 PDT 2019
+//    Add tetrahedralize option.
 // ****************************************************************************
 
 avtVTKWriter::avtVTKWriter(DBOptionsAttributes *atts) :stem(), meshName(), fileNames()
 {
     doBinary = false;
     doXML = false;
     nblocks = 0;
+    tetrahedralize = false;
```

Comment:
Does this need to be set false here since it is reset on line 119?

Reply:
I removed that too.